### PR TITLE
Kulrice 14274

### PR DESCRIPTION
**Pull request by jruch**
Move sensitive data from the Incident form to the user session when mode is production.  The formatting isn't displaying correctly in github, but is correct in the IDE.

### CODE REVIEW

File: rice-middleware/kns/src/main/java/org/kuali/rice/kns/web/struts/action/KualiExceptionHandlerAction.java [hunk 1]
```diff
@@ -176,9 +198,38 @@ protected ActionForward processException(ActionMapping mapping,
...
 		Map<String, String> properties = (Map<String, String>) request
 				.getAttribute(IncidentReportUtils.EXCEPTION_PROPERTIES);
-		// Construct the exception incident object
+
+        // Construct the exception incident object
 		KualiExceptionIncident ei = reporterService.getExceptionIncident(
 				exception, properties);
+
+        // For Production, hide sensitive data in user session
+        if(!ConfigContext.getCurrentContextConfig().getDevMode()) {
```
Comment (cniesen) [inline rice-middleware/kns/src/main/java/org/kuali/rice/kns/web/struts/action/KualiExceptionHandlerAction.java]: Why use the user session only for production?  Using the same logic (i.e. storing it always in the user session) will make sure that it has been tested during development.

**Comment by cniesen** [comment]
If you're updating the code, doing a squash to get rid of the ide changes would be nice.

File: rice-middleware/kns/src/main/java/org/kuali/rice/kns/web/struts/action/KualiExceptionHandlerAction.java (hunk 1 above)
Comment (cniesen) [inline rice-middleware/kns/src/main/java/org/kuali/rice/kns/web/struts/action/KualiExceptionHandlerAction.java]: Probably should be using ConfigContext.getCurrentConfig.isProductionEnvironment()

Comment (jruch) [inline rice-middleware/kns/src/main/java/org/kuali/rice/kns/web/struts/action/KualiExceptionHandlerAction.java]: I agree.  No point in having separate logic.  Will take out, and then no need for checking for a production environment.

**Comment by cniesen** [comment]
retest this please

**Comment by cniesen** [comment]
retest this please

**Comment by cniesen** [comment]
retest this please

**Pull request merged**